src/ec/packages/ec-component-site-header-core/ecl-site-header-core.html.twig

```diff
@@ -179,7 +211,8 @@
                 name: 'logged-in',
                 size: 's'
               },
-              extra_classes: 'ecl-site-header-core__icon'
+              extra_classes: 'ecl-site-header-core__icon',
```

Comment:
mmh..the diff here is about a line where the trailing comma has been added since it is now followed by ` _compliance_: _compliance`
In general trailing commas in object literals are not used in these templates, if we want to move to this approach we can maybe do it at once in a separated ticket.

Reply:
It was just a suggestion of course.